src/character/CharacterHandler.cpp

```diff
@@ -175,13 +175,33 @@ void CharacterHandler::do_create(std::uint32_t account_id, std::uint32_t realm_i
 	if(items != dbc_.char_start_outfit.end()) {
 		populate_items(character, items->second);
 	} else { // could be intentional, so we'll keep going
-		LOG_DEBUG(logger_) << "No starting item data found for race" <<
+		LOG_DEBUG(logger_) << "No starting item data found for race " <<
 			character.race << ", class " << character.class_ << LOG_ASYNC;
 	}
 
-	// populate spells information
+	// populate starting spells
+	auto spells = std::find_if(dbc_.char_start_spells.begin(), dbc_.char_start_spells.end(), [%](auto& record ) {
+		return record.second.race_id == character.race && record.second.class__id == character.class_;
+	});
+
+	if (spells != dbc_.char_start_spells.end()) {

```

Comment:
Slight formatting change.
```suggestion
	if(spells != dbc_.char_start_spells.end()) {
```